### ISSUE TITLE
[RFE] remediate role support for RHEL 7

### ISSUE THREAD

**Issue by sahara101** [redhat-cop/infra.leapp#184]
Hi!

any chance to use remediation on rhel7?

Thanks!

**Comment by sahara101** [comment]
I also get this:

ERROR! the role 'infra.leapp.remediate' was not found in /etc/ansible/books/roles:/etc/ansible/roles:/etc/ansible/books

The error appears to be in '/etc/ansible/books/remediate.yml': line 18, column 15, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

      ansible.builtin.import_role:
        name: infra.leapp.remediate
              ^ here

**Comment by djdanielsson** [comment]
I do not believe there has been a release since remediate role was added, and I am pretty sure it states that it only supports RHEL8

**Comment by sahara101** [comment]
Yes, true. I was asking if you plan to support 7to8
Also, I cannot understand why I get that error? That one should not have anything to do with rhel version?

Thanks!

**Comment by djdanielsson** [comment]
because we have not done a release yet that includes remediation so unless you are pulling from git to build the collection, it is saying it is missing because it doesn't exist in the current 1.1.0 version.

**Comment by sahara101** [comment]
sounds good, thanks A LOT!

**Comment by bonnevil** [comment]
The current version of the collection still says that remediation is not supported for migrations from RHEL 7 to RHEL 8.  This is an issue because RHEL 7 just reached EOL and I think there's a lot of folks interested in automated remediation of issues from pre-upgrade reports for RHEL 7 to 8 upgrades.
